src/app/modules/submission/encrypt-submission/encrypt-submission.constants.ts

```diff
@@ -0,0 +1 @@
+export const newEncryptionBoundaryFlag = 'newEncryptionBoundary'
```

Comment:
/nit: uppercase for const, i.e. NEW_ENCRYPTION_BOUNDARY_FLAG

Reply:
Oh, with the form schema typing I can remove this constant already. Will do so in a bit!

Reply:
Constant removed in [28ac14c](https://github.com/opengovsg/FormSG/pull/6595/commits/28ac14c6e39d962eb83e168fe8290f9b6783a53e)!